### PR TITLE
Add user portal migration scripts

### PR DESCRIPTION
**Pull request by pamodaaw**
During the migration,

* User portal configurations will be renamed as **My Account**
* Two new OIDC properties will be defined for the application 
-- tokenRevocationWithIDPSessionTermination=true
-- tokenBindingValidatio = true
* Token binding type is changed as `sso-session`
-- tokenBindingType = sso-session
* Since "My Account" is a SaaS application, the user portal configurations of the tenant other than the super tenant will be deleted

### CODE REVIEW

File: components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/postresql.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+UPDATE UM_HYBRID_ROLE 
+SET 
+	UM_ROLE_NAME = 'Application/My Account' 
+WHERE 
+	UM_ROLE_NAME = 'Application/User Portal' AND UM_TENANT_ID= -1234;
+    
+DELETE FROM UM_HYBRID_ROLE WHERE UM_ROLE_NAME = 'Application/User Portal' AND UM_TENANT_ID <> -1234;
+
```
Comment (Buddhimah) [inline components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/postresql.sql:8]: redundant new line

File: components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/mysql_cluster.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+UPDATE UM_HYBRID_ROLE 
+SET 
+	UM_ROLE_NAME = 'Application/My Account' 
+WHERE 
+	UM_ROLE_NAME = 'Application/User Portal' AND UM_TENANT_ID= -1234;
+    
+DELETE FROM UM_HYBRID_ROLE WHERE UM_ROLE_NAME = 'Application/User Portal' AND UM_TENANT_ID <> -1234;
+
```
Comment (Buddhimah) [inline components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/mysql_cluster.sql:8]: redundant new line

File: components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/mssql.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+UPDATE UM_HYBRID_ROLE 
+SET 
+	UM_ROLE_NAME = 'Application/My Account' 
+WHERE 
+	UM_ROLE_NAME = 'Application/User Portal' AND UM_TENANT_ID= -1234;
+    
+DELETE FROM UM_HYBRID_ROLE WHERE UM_ROLE_NAME = 'Application/User Portal' AND UM_TENANT_ID <> -1234;
+
```
Comment (Buddhimah) [inline components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/mssql.sql:8]: redundant new line

**Review by pulasthi7 — APPROVED** [review]

File: components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/mssql.sql (hunk 3 above)
Comment (pamodaaw) [reply components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/mssql.sql]: fixed

File: components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/mysql_cluster.sql (hunk 2 above)
Comment (pamodaaw) [reply components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/mysql_cluster.sql]: fixed

File: components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/postresql.sql (hunk 1 above)
Comment (pamodaaw) [reply components/org.wso2.is.migration/migration-resources/5.11.0/dbscripts/step4/um/postresql.sql]: fixed

File: components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/migrator/UserPortalDataMigrator.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.carbon.identity.core.migrate.MigrationClientException;
+import org.wso2.carbon.is.migration.service.SchemaMigrator;
+
+/**
+ * This class handles the migration of the user-portal to the new application My Account.
+ */
+public class UserPortalDataMigrator extends SchemaMigrator {
```
Comment (ashensw) [inline components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/migrator/UserPortalDataMigrator.java:29]: Can't we use the SchemaMigrator itself as we are not doing any other processing here?

Comment (pamodaaw) [reply components/org.wso2.is.migration/migration-service/src/main/java/org/wso2/carbon/is/migration/service/v5110/migrator/UserPortalDataMigrator.java:29]: There is a requirement to not to migrate the user-portal to my account. In that case, having a separate migrator is more readable and easily configurable.

**Pull request merged**